backend/src/v5/services/sso/frontegg/components/accounts.js

```diff
@@ -179,6 +202,25 @@ Accounts.addUserToAccount = async (accountId, email, name, emailData) => {
 	}
 };
 
+Accounts.getClaimedDomains = async (accountId) => {
+	try {
+		const config = await getConfig();
+		const headers = {
+			...await getBearerHeader(),
+			[HEADER_TENANT_ID]: accountId,
+		};
+
+		const { data } = await get(`${config.vendorDomain}/team/resources/sso/v1/configurations`, headers);
+		return data.flatMap(({ enabled, domains }) => {
+			if (!enabled) return [];
+			return domains.map(({ domain }) => domain);
+		});
+	} catch (err) {
+		logger.logError(`Failed to get claimed domains within an account: ${JSON.stringify(err?.response?.data)} `);
```

Comment:
just in case it fails to get the domains cant we just return [ ] ? cause in this case the app will crash

Reply:
how did you trigger an app crash? 

This should never happen.

None the less I'll add a catch case on removeFronteggSessionIfNeeded and return a 500.